version/compatibility.json

```diff
@@ -1,5 +1,6 @@
 {
   "26": [
+    "v1.1.2",
```

Comment:
have we missed the in-between version of 1.1.1?

Reply:
unfortunately, I didn't update version and compatibility in code in cortina 1 PR, so we can't have them here
but starting form cortina 2, we'll keep this stuff consistent